tests/resources/test_comercial_documents.py

```diff
@@ -0,0 +1,57 @@
+import os
+from datetime import date
+
+import pytest
+from cuenca_validations.types import DocumentType
+from cuenca_validations.types.requests import DocumentRequest
+from pydantic import ValidationError
+
+from cuenca import ComercialDocument
+
+
+@pytest.mark.vcr
+def test_comercial_documents_create():
+    document = ComercialDocument.create(
+        DocumentRequest(
+            client_name='Iron Man',
+            clabe='002000000000000008',
+            address='Address Foo',
+            rfc='GODE561231GR8',
+            date=(2020, 11),
+            document_type=DocumentType.invoice,
+        )
+    )
+    assert document.id is not None
+    assert document.body is not None
+
+    zip_name = 'CDceVNfKmzRH6PDgFnlHwgsg.zip'
+    base_dir = os.path.abspath(os.getcwd())
+
+    document.download(path=f'{base_dir}/tests/resources')
+
+    assert os.path.exists(f'{base_dir}/tests/resources/{zip_name}')
```

Comment:
@matin because of this, delete after test

Reply:
Got it. It's better practice to return some form of a readable io. If the consumer of the method wants to write it to the FS, that's up to them. They may want to write it to S3 instead.